test/Riok.Mapperly.Tests/TestSourceBuilderOptions.cs

```diff
@@ -27,6 +28,8 @@ public record TestSourceBuilderOptions(
     public static readonly TestSourceBuilderOptions WithDeepCloning = new(UseDeepCloning: true);
     public static readonly TestSourceBuilderOptions WithReferenceHandling = new(UseReferenceHandling: true);
 
+    public static readonly TestSourceBuilderOptions WithDescParametersConstructorPriority = new(PreferParameterlessConstructors: false);
```

Comment:
Nitpick: IMO an easier to read and understandable name would be `PreferParameterizedConstructors` (same with the added test method name)

Reply:
done